src/libs/Launcher.functions.ts

```diff
@@ -0,0 +1,27 @@
+import CozyClient, { Q } from 'cozy-client'
+import { CozyClientDocument } from 'cozy-client/types/types'
+
+export const cleanExistingAccountsForKonnector = async (
+  client: CozyClient,
+  konnectorSlug: string,
+  log?: MiniLogger
+): Promise<void> => {
+  const { data: accounts } = (await client.query(
+    Q('io.cozy.accounts').where({
+      accountType: konnectorSlug
+    })
+  )) as { data: CozyClientDocument[] }
+
+  if (accounts.length === 0) {
+    log?.info(`No existing accounts for "${konnectorSlug}"`)
+    return
+  }
+
+  log?.info(
+    `Deleting ${accounts.length} existing accounts for "${konnectorSlug}"`
+  )
+
+  for (const account of accounts) {
+    await client.destroy(account)
```

Comment:
Should we also delete triggers and jobs linked to those accounts?

Reply:
as far as I understood they should be deleted by the stack (or if they aren't it's a stack issue). cc @Crash--

Reply:
I confirm.